messaging.rst

```diff
@@ -276,6 +276,147 @@ This should match `the Secret message in encoding/messages file <https://github.
 |                      |                        | is the hash of the whole message                           |
 +----------------------+------------------------+------------------------------------------------------------+
 
+.. _withdraw-request-message:
+
+Withdraw Request
+-------------
+
+Message used by the a channel participant node to request the other participant signature on a new increased ``total_withdraw`` value.
+
+Preconditions
+^^^^^^^^^^^^^
+
+- The channel for which the withdraw is requested should be open.
+- The ``total_withdraw`` value should only ever increase.
+- The participant's channel balance should be larger or equal to ``withdraw_amount``,
+  which is calculated using ``new_total_withdraw - previous_total_withdraw``.
+- The new total_withdraw value should not cause an overflow.
+- The message should be sent by one of the channel participants.
+- The :term:`nonce` is increased by ``1`` with respect to the previous :term:`nonce`
+- The ``signature`` must be from the :term:`sender` of the request.
+
```

Comment:
Is `sender == participant` required?  I'd mention that here

Reply:
Done